azext_edge/tests/edge/support/test_support_unit.py

```diff
@@ -897,59 +909,6 @@ def test_get_bundle_path(mocked_os_makedirs):
 
 
 # TODO - @elsie4ever
-# # TODO - test zipfile write for specific resources
-# # MQ connector stateful sets need labels based on connector names
-# @pytest.mark.parametrize(
-#     "mocked_cluster_resources",
-#     [[MQTT_BROKER_API_V1B1]],
-#     indirect=True,
-# )
-# @pytest.mark.parametrize(
-#     "custom_objects",
-#     [
-#         # connectors present
-#         {
-#             "items": [
-#                 {
-#                     "metadata": {"name": "mock-connector", "namespace": "mock-namespace"},
-#                 },
-#                 {
-#                     "metadata": {"name": "mock-connector2", "namespace": "mock-namespace2"},
-#                 },
-#             ]
-#         },
-#         # no connectors
-#         {"items": []},
-#     ],
-# )
-# def test_mq_list_stateful_sets(
```

Comment:
This is covered in other tests now ?

Reply:
Hi Paymaun, sorry forgot mentioned this, this test_mq_list_stateful_sets was testing for connector resource, but since now connector is removed from broker, this test should be removed as well